modular_zubbers/code/modules/silicons/silicon.dm

```diff
@@ -0,0 +1,62 @@
+/mob/living/silicon/Topic(href, href_list)
+	. = ..()
+	if(href_list["open_door"])
+		if(!isAI(src))
+			to_chat(src, span_warning("You shouldn't be seeing this bug a coder"))
```

Comment:
I got this ingame while I was deployed to my shell.

Reply:
fixed